cell.go

```diff
@@ -48,6 +48,12 @@ func (cb *CellBuffer) SetContent(x int, y int,
 	if x >= 0 && y >= 0 && x < cb.w && y < cb.h {
 		c := &cb.cells[(y*cb.w)+x]
 
+		if len(combc) > 0 {
+			cpy := make([]rune, len(combc))
+			copy(cpy, combc)
+			combc = cpy
+		}
+
```

Comment:
Instead of this approach, I think it would be better to make a copy a different way.  Specifically:

c.currComb = append([]rune{}, combc...)

This would keep the code a bit more compact, and I think is easier to grok at a glance.

Reply:
You're right. That's more elegant. I didn't realize that `append(nil, nil...) == nil`.

I will change it.